seed_helen_comments.py

```diff
@@ -0,0 +1,354 @@
+        for l in html_body:
+            match = re.match(r'.*(http:.*/_(?!ARC|AG|UMB).*.jpg).*', l)
+            # match = re.match(r'.*(http:\/\/assets.vogue.com\/photos\/.*\/master\/pass\/.*.[jpg|JPG]).*', l)
+            if match:
+                image_urls.add(match.group(1))
+    # print "image urls:", image_urls
+    # print len(image_urls)
+    # image_urls = list(image_urls)
+    return list(image_urls)
+    # pillow_loop(image_urls)
+
+
+def feed_urls():
+    # main fxn to call url generator and aggregate top colors by show
+    generated_urls = {year: {season: {brand: 'http://www.vogue.com/fashion-shows/{}-{}-ready-to-wear{}'.format(season, year, brand_url) for brand, brand_url in brands.items()} for season in seasons} for year in years}
+    # print "generated urls:", generated_urls
+    for year in generated_urls.keys():
+        for season in generated_urls[year].keys():
+            for brand, brand_url in generated_urls[year][season].items():
+                print "generated urls", generated_urls
+                print "brand url:", brand_url
+                print "season", season
+                print "brand name:", brand
+                show = load_show(year, season, brand)
+                image_urls = img_urls(brand_url)
+
+                top_show_colors = []
+                for url in image_urls:
+                    final_named_colors = pillow_loop(url)
+                    for color in final_named_colors:
+                        top_show_colors.append(color)
+                print "at feed urls, top show colors: ", top_show_colors
+
+                count_colors = {color: top_show_colors.count(color) for color in top_show_colors}
+                top_show_colors = sorted(set(top_show_colors), key=count_colors.get, reverse=True)
+                top_show_colors = top_show_colors[:10]
+                load_show_colors(top_show_colors, brand, year, season)
+                # jus have to pass top show color & show (or just show.show_id)
+
+
+def load_brands(brands):
+    """Load brands into database."""
+
+    print "Brands"
+
+    for key, value in brands.items():
+        brand_name = key
+
+        brand = Brand(brand_name=brand_name)
+
+    # We need to add to the session or it won't ever be stored
+        db.session.add(brand)
+
+    # Once we're done, we should commit our work
+    db.session.commit()
+
+
+def load_colors():
+    """Load colors/hex from css3 dict into database."""
+
+    print "Color"
+
+    for key, value in css3_hex_to_names.items():
+        color_hex, color_name = key, value
+        color = Color(color_hex=color_hex,
+                      color_name=color_name)
+
+        db.session.add(color)
+
+    # Once we're done, we should commit our work
+    db.session.commit()
+
+
+def load_show(year, season, brand):
+    """Load shows into database."""
+
+    print "Show"
+
+    brand_id = db.session.query(Brand).filter_by(brand_name=brand).one().brand_id
+
+    year = 2017
```

Comment:
Looks like 2017 is a constant you're using in multiple places. If you defined it as a global comment like I suggested above, this would be `YEARS[0]` (or maybe just `YEAR` if you're only using 1?), and you would only have to change it to one place if you switch to a different year.